docs/plaindate.md

```diff
@@ -301,7 +304,8 @@ For the ISO 8601 calendar, this is 365 or 366, depending on whether the year is
 Usage example:
 
 ```javascript
-date = Temporal.now.plainDate();
+({ calendar } = Intl.DateTimeFormat().resolvedOptions());
```

Comment:
I understand that this example actually should work in any calendar, but we did learn that `({ calendar } = Intl.DateTimeFormat().resolvedOptions())` isn't a foolproof way to get the user's preferred calendar because "the user's preferred calendar" is an ill-defined thing in the first place. So I would rather that the documentation not present that as the correct way to write cross-calendar code. I'd go with `plainDateISO()` in these examples.